panoramapublic/resources/queries/panoramapublic/peptideSearchResults.sql

```diff
@@ -0,0 +1,25 @@
+PARAMETERS
+(
+    peptideSequence VARCHAR,
+    exactMatch BIT default false
+)
+
+SELECT trn.FileName,
+       ex.Instrument,
+       p.ModifiedPeptideDisplayColumn,
+       p.CalcNeutralMass,
+       p.PeptideGroupId.Label,
+       ex.created,
+       p.Sequence,
+       ex.title,
+       ex.organism,
+       ex.citation,
+       ex.pxid,
+       peptideSequence AS peptideSequence,
```

Comment:
I think this also needs to select exactMatch so that it's available to render the URL in the DisplayColumn

Reply:
this is the query after the navigation from search page. Display column is in the previous page rendering peptideResults.sql. I think I should remove line 18 from here.